aws/config.go

```diff
@@ -143,6 +144,7 @@ type Config struct {
 	KinesisAnalyticsEndpoint string
 	KmsEndpoint              string
 	LambdaEndpoint           string
+	RamEndpoint              string
```

Comment:
Since we're not wiring up endpoint customization in the provider configuration (see `aws/provider.go`) we should either implement it or remove it here, e.g. removing this line, `awsRamSess` and using `client.ramconn = ram.New(sess)` 👍

Reply:
This still needs to be addressed either which way. 😄